README.md

```diff
@@ -4,9 +4,9 @@ The platform assists member in managing expenses, tasks , voting, and more.
 
 ## ERD
 
-![ERD](https://user-images.githubusercontent.com/74178515/204143707-970216f5-e382-47d1-ab2e-209aa7aaa18b.png)
+![ERD](https://lucid.app/documents/embedded/af373954-3144-4325-9253-494a0af1512b?invitationId=inv_d92b11b0-d0dd-445e-98c2-8493166012ea#)
```

Comment:
I wonder if the Commune object should have two different chore fields: one for chores that are passed or ones that are still in voting

Reply:
I don't think so but let's check what the rest of the team thinks
@FaezZangariya @ofirtzabari @EliasAndry